tests/e2e/cucumber/features/smoke/share.ocis.feature

```diff
@@ -7,40 +7,50 @@ Feature: share
       | Brian |
 
   Scenario: folder
-    When "Alice" logs in
+    Given "Alice" creates the following folder in personal space using API
+      | name                   |
+      | folder_to_shared       |
+      | folder_to_customShared |
+    And "Alice" logs in
     And "Alice" opens the "files" app
-    And "Alice" creates the following resources
-      | resource         | type   |
-      | folder_to_shared | folder |
     And "Alice" uploads the following resource
-      | resource  | to               |
-      | lorem.txt | folder_to_shared |
+      | resource      | to                     |
+      | lorem.txt     | folder_to_shared       |
+      | lorem-big.txt | folder_to_customShared |
     When "Alice" shares the following resource using the sidebar panel
-      | resource         | recipient | type | role   |
-      | folder_to_shared | Brian     | user | editor |
+      | resource               | recipient | type | role   |
+      | folder_to_shared       | Brian     | user | editor |
+      | folder_to_customShared | Brian     | user | editor |
     And "Brian" logs in
     And "Brian" opens the "files" app
     And "Brian" navigates to the shared with me page
     And "Brian" accepts the following share
-      | name             |
-      | folder_to_shared |
+      | name                   |
+      | folder_to_shared       |
+      | folder_to_customShared |
     And "Brian" renames the following resource
       | resource                   | as            |
       | folder_to_shared/lorem.txt | lorem_new.txt |
     And "Brian" uploads the following resource
       | resource   | to               |
       | simple.pdf | folder_to_shared |
-    When "Alice" opens the "files" app
+    And "Alice" opens the "files" app
     And "Alice" uploads the following resource
       | resource          | to               | option  |
       | PARENT/simple.pdf | folder_to_shared | replace |
     And "Brian" downloads old version of the following resource
       | resource   | to               |
       | simple.pdf | folder_to_shared |
-    When "Brian" restores following resources
+    And "Brian" restores following resources
       | resource   | to               | version |
       | simple.pdf | folder_to_shared | 1       |
-    When "Alice" deletes the following resources
+    And "Alice" removes following sharee
+      | resource               | recipient |
+      | folder_to_customShared | Brian     |
+    Then "Brian" should not be able to see the following shares
+      | resource               | owner        |
+      | folder_to_customShared | Alice Hansen |
+    And "Alice" deletes the following resources
```

Comment:
```suggestion
    When "Alice" deletes the following resources
```
this when step is not checked if was successful or not.

Reply:
not needed in e2e

Reply:
@ScharfViktor @amrita-shrestha why not needed in e2e? what is the purpose of running the step without checking its effect :thinking:?

Reply:
I suggest to add step on the end of the scenario:
```
Then "Brian" should not be able to see the following shares
      | resource               | owner        |
      | folder_to_shared | Alice Hansen |
      ```

as result will be:
When Alice removes share folder1
Then Brian should not be able to see folder1
When Alice deletes folder2
Then Brian should not be able to see folder2